src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsadjudicationsinsightsapi/config/OpenApiConfiguration.kt

```diff
@@ -90,15 +81,4 @@ class OpenApiConfiguration(
       }
     }
   }
-
-  private fun createErrorApiResponse(message: String): ApiResponse {
```

Comment:
Raised [NN-5326](https://dsdmoj.atlassian.net/browse/NN-5326) to handle errors better, at which point we can use the Swagger annotation `@Operation` combined with `schema = io.swagger.v3.oas.annotations.media.Schema(implementation = ErrorResponse::class)`